src/lib/wait.c

```diff
@@ -41,8 +42,8 @@ int wait_for_completion_timeout(completion_t *comp)
 	}
 
 	/* timeout */
-	trace_error_value(c->timeout);
-	trace_error_value(c->complete);
+	trace_error(TRACE_CLASS_WAIT, "wait_for_completion_timeout, timeout: " /
```

Comment:
No need for "/" at the end of the line (also compiler error :) )

Reply:
PRIu64 is something like "%llu" AKA a string.

Reply:
ok